src/lib/gw2/itemStats.js

```diff
@@ -0,0 +1,90 @@
+import { queryItemAttributes } from 'gw2itemstats';
```

Comment:
I think that initially we can add it as a library and later on move it to server side, that's what i meant above, i know its pretty big, eventually this library is just a json object wrapped in a getter function